frontend/packages/dev-console/src/components/import/advanced/ServerlessScalingSection.tsx

```diff
@@ -40,6 +49,21 @@ const ServerlessScalingSection: React.FC = () => {
           'devconsole~Limits the amount of concurrent requests allowed into one instance of the Application at a given time (hard limit), and is configured in the revision template. If not specified, will be defaulted to the value set in the Cluster config.',
         )}
       />
+      <NumberSpinnerField
+        name="serverless.scaling.concurrencyutilization"
+        label={t('devconsole~Concurrency utilization')}
+        helpText={t('devconsole~Percentage of concurrent requests utilization before scaling up.')}
+      />
+      <ResourceLimitField
+        name="serverless.scaling.autoscale.autoscalewindow"
+        label={t('devconsole~Autoscale window')}
+        unitName="serverless.scaling.autoscale.autoscalewindowUnit"
+        unitOptions={AutoscaleWindowUnits}
+        defaultUnitSize={`${autoscale.defaultAutoscalewindowUnit}`}
+        helpText={t(
+          'devconsole~Duration to look back for making auto-scaling decisions. The service is scaled to zero if no request was received in during that time.',
+        )}
```

Comment:
Have we taken this help text from configmap, would be good to get it reviewed with UXD

Reply:
@invincibleJai copy looks good. 

One spacing request. Can we move the help text for the last field, Autoscale window, closer to the field? It looks like it is not associated with that field. Besides that, it looks great!

Reply:
@beaumorley remove the extra spacing 

![image](https://user-images.githubusercontent.com/2561818/107803436-9227e500-6d88-11eb-9072-9162687c6b51.png)

Reply:
@vikram-raj Thank you. LGTM.